samples/bluetooth/llpm/README.rst

```diff
@@ -57,7 +57,7 @@ GATT Latency Service
      - BT_GATT_CHRC_WRITE, BT_GATT_PERM_WRITE
 
 This sample transmits data between two development kits to measure the transmission latency in between.
-One of the devices is connected as a *master* and another is connected as a *slave*.
+One of the devices is connected as a *central* and another is connected as a *peripheral*.
```

Comment:
Aren't central and peripheral capitalized in the Bluetooth context?

Reply:
I had a quick look through the rest of the documentation, but could not find a consistent use of it. Therefore I kept the format as is